### PR TITLE
Add support for CIS Microsoft 365 Foundations Benchmark v3.0.0 - [09-29-2023] closes #56

### LINKED ISSUE #56: Add support for CIS Microsoft 365 Foundations Benchmark  v3.0.0 - [09-29-2023]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by khushboo9024**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

**Comment by rajlearner17** [comment]
@khushboo9024 Pls add 
Section 3.1 | 3.1.1 (L1) Ensure Microsoft 365 audit log search is Enabled (Earlier v200_5_2)

File: cis_v300/section_1.sp [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    cis_type              = "automated"
+    microsoft_365_license = "E3"
+    service               = "Azure/ActiveDirectory"
+  })
+}
+
+control "cis_v300_1_3_3" {
+  title         = "1.3.3 Ensure 'External sharing' of calendars is not available"
+  description   = "External calendar sharing allows an administrator to enable the ability for users to share calendars with anyone outside of the organization. Outside users will be sent a URL that can be used to view the calendar."
+  query         = query.microsoft365_calendar_sharing_disabled
```
Comment (rajlearner17) [inline cis_v300/section_1.sp:142]: @khushboo9024 Pls validate this 

```
1.3.3 Ensure 'External sharing' of calendars is not available .........................................................................................      
| |     ERROR: relation "microsoft365_my_calendar" does not exist (SQLSTATE 42P01)
```

Comment (khushboo9024) [reply cis_v300/section_1.sp:142]: @rajlearner17 Please update the plugin and try again.

File: cis_v300/section_5.sp [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+    cis_item_id           = "5.2.2.8"
+    cis_level             = "1"
+    cis_type              = "manual"
+    microsoft_365_license = "E3"
+    service               = "Azure/ActiveDirectory"
+  })
+}
+
+benchmark "cis_v300_5_2_4" {
+  title         = "5.2.4 Password reset"
```
Comment (madhushreeray30) [inline cis_v300/section_5.sp:318]:
```suggestion
  title = "5.2.4 Password reset"
```

File: cis_v300/section_5.sp [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+    cis_item_id           = "5.2.4.1"
+    cis_level             = "1"
+    cis_type              = "manual"
+    microsoft_365_license = "E3"
+    service               = "Azure/ActiveDirectory"
+  })
+}
+
+benchmark "cis_v300_5_2_6" {
+  title         = "5.2.6 Risky activities"
```
Comment (madhushreeray30) [inline cis_v300/section_5.sp:345]:
```suggestion
  title = "5.2.6 Risky activities"
```

**Review by madhushreeray30 — COMMENTED** [review]
@khushboo9024 Please take a look at the review comments, thanks!

**Review by madhushreeray30 — APPROVED** [review]
LGTM 👍

**Review by madhushreeray30 — APPROVED** [review]

**Pull request merged**